Kernel/Output/HTML/Layout.pm

```diff
@@ -93,6 +93,7 @@ sub new {
     # allocate new hash for object
     # %Param often has a entry for 'SetCookies'
     my $Self = bless {%Param}, $Type;
+    my $HasRichTextEditor = $Self->{'Frontend::RichText'};
```

Comment:
The sysconfig setting is the general information that rich text editor is activated. But we need the information specific for a frontend module.

Reply:
Would my $HasRichTextEditor = $Self->{'Frontend::RichText::CKEditor'} be more specific?

Reply:
The specifity is not the issue. OTOBO has a pretty elaborate configuraion system. Take a look at https://github.com/RotherOSS/otobo/blob/f8e9f730af50ee821fbf88f3eff033b706c28604/Kernel/Config/Files/XML/Framework.xml#L376 . The setting Frontend::RichText simply indicates that support for a rich text editor is active. But for all practical purposes, RTE is always enabled.  So this information is not really interesting.

Perhaps it is easier when actually looking at the installation. You can go to https://otobo.de/en/open-source-ticketing-system/#demos and log in as 'Lena/Lena'. You can change the user language at top left. Then you can choose Tickets->New Phone Ticket  to get to a page that has an embedded editor. Looking at the URL you see "Action=AgentTicketPhone". This gives you the information that the frontend module _Kernel/Modules/AgentTicketPhone.pm_ needs to set the flag 'HasRichTextEditor'.

Reply:
Would it be something like this my $HasRichTextEditor = $ConfigObject->Get("Frontend::$Self->{Action}")?

Reply:
No, trial and error is not productive here. 

Perhaps it is better to move away from the actual changes and get a consensus of what should be achieved. Could you go back to #3223 and describe in your own words what should be achieved and how you plan to go ahead with it? In this way we can achieve a common understanding. But please don't despair when you are overwhelmed. OTOBO has an history of over 20 years and things are sometimes a bit peculiar in OTOBO.